views/address.tmpl

```diff
@@ -17,7 +17,7 @@
                 <table class="table table-sm striped">
                     <thead>
                         <th>Transactions ID</th>
-                        <th>Total Value</th>
+                        <th>DCR Value</th>
```

Comment:
I kind of liked Total, because it implied it was a sum over the transactions paying to the address.  Perhaps Total DCR Value or Total Value (DCR) if that's not to much space?